package.json

```diff
@@ -42,8 +42,10 @@
   "devDependencies": {
     "eslint": "^0.15.0",
     "expect.js": "^0.3.1",
-    "istanbul": "^0.3.6",
-    "jsdom": "^4.0.1",
+    "karma": "^0.12.31",
+    "karma-coverage": "^0.2.7",
+    "karma-mocha": "^0.1.10",
+    "karma-phantomjs-launcher": "^0.1.4",
```

Comment:
Not sure if now is the time or not but you could add additional launchers for test runs in CI.